.github/workflows/e2e.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           TESTFLAGS: -coverprofile=/tmp/coverage/coverage.txt
       -
         name: Send to Codecov
-        uses: codecov/codecov-action@v4
+        uses: codecov/codecov-action@v5
         with:
-          file: ./build/coverage/coverage.txt
+          files: ./build/coverage/coverage.txt
```

Comment:
`Files` takes a comma-separated list.. bit of an odd choice for this (not sure why they didn't pick an array / slice)